webrtc/RTCTrackEvent-constructor.html

```diff
@@ -0,0 +1,171 @@
+    });
+
+    assert_equals(trackEvent.receiver, receiver);
+    assert_equals(trackEvent.track, track);
+    assert_array_equals(trackEvent.streams, [stream]);
+    assert_equals(trackEvent.transceiver, transceiver);
+
+  }, `new RTCTrackEvent() with valid receiver, track, streams, transceiver should succeed`);
+
+  test(t => {
+    const pc = new RTCPeerConnection();
+    const transceiver = pc.addTransceiver('audio');
+    const { receiver } = transceiver;
+    const { track } = receiver;
+
+    const stream1 = new MediaStream([track]);
+    const stream2 = new MediaStream([track]);
+
+    const trackEvent = new RTCTrackEvent('track', {
+      receiver, track, transceiver,
+      streams: [stream1, stream2]
+    });
+
+    assert_equals(trackEvent.receiver, receiver);
+    assert_equals(trackEvent.track, track);
+    assert_array_equals(trackEvent.streams, [stream1, stream2]);
+    assert_equals(trackEvent.transceiver, transceiver);
+
+  }, `new RTCTrackEvent() with valid receiver, track, multiple streams, transceiver should succeed`);
+
+  test(t => {
+    const pc = new RTCPeerConnection();
+    const transceiver = pc.addTransceiver('audio');
+    const receiver = pc.addTransceiver('audio').receiver;
+    const track =  pc.addTransceiver('audio').receiver.track;
+
+    const stream = new MediaStream();
+
+    const trackEvent = new RTCTrackEvent('track', {
+      receiver, track, transceiver,
+      streams: [stream]
+    });
+
+    assert_equals(trackEvent.receiver, receiver);
+    assert_equals(trackEvent.track, track);
+    assert_array_equals(trackEvent.streams, [stream]);
+    assert_equals(trackEvent.transceiver, transceiver);
+
+  }, `new RTCTrackEvent() with unrelated receiver, track, streams, transceiver should succeed`);
+
+  test(t => {
+    const pc = new RTCPeerConnection();
+    const transceiver = pc.addTransceiver('audio');
+    const { receiver } = transceiver;
+    const { track } = receiver;
+
+    assert_throws(new TypeError(), () =>
+      new RTCTrackEvent('track', {
+        receiver, track
+      }));
+
+  }, `new RTCTrackEvent() with no transceiver should throw TypeError`);
+
+  test(t => {
+    const pc = new RTCPeerConnection();
+    const transceiver = pc.addTransceiver('audio');
+    const { receiver } = transceiver;
+
+    assert_throws(new TypeError(), () =>
+      new RTCTrackEvent('track', {
+        receiver, transceiver
+      }));
+
+  }, `new RTCTrackEvent() with no track should throw TypeError`);
+
+  test(t => {
+    const pc = new RTCPeerConnection();
+    const transceiver = pc.addTransceiver('audio');
+    const { receiver } = transceiver;
+    const { track } = receiver;
+
+    assert_throws(new TypeError(), () =>
+      new RTCTrackEvent('track', {
+        track, transceiver
+      }));
+
+  }, `new RTCTrackEvent() with no receiver should throw TypeError`);
+
+  /*
+    Coverage Report
+      Tested        14
+      Not Tested     0
+      Untestable     0
+      Total         14
```

Comment:
Hmm strictly speaking the lines should be applicable to the `ontrack` tests instead of this constructor test. Since there is no explicit steps for the constructor, this can probably be counted as trivial test and marked as 1 tested.